### PR TITLE
Traitor uplink items rework

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request
Lots of changes intended to spice up the meta and increase the diversity of traitor items bought again.

Full list of changes:
- Taeclowndo shoes can be used by anyone now, but still only bought by the Clown if not discounted.
- Taeclowndo shoes decreased from 14 to 12 TC
- Dart Gun decreased from 4 to 3 TC
- Boxed Origami Kit decreased from 8 to 6 TC
- Karate Scroll decreased from 6 to 4 TC
- Pizza Box Bomb decreased from 5 to 3 TC
- Codespeak Manual decreased from 3 to 2 TC
- Syndicate Tome decreased from 5 to 2 TC
- Syndicate Surgery Duffel Bag decreased from 3 to 2 TC
- Energy Sword decreased from 10 to 8 TC
- Syndicate Bomb increased from 11 to 12 TC
- Poison Kit increased from 6 to 7 TC
- Sleepy Pen increased from 4 to 5 TC
- Contract Kit random starting items list changes:
  - Removed: Magic eightball, EMP flashlight, uplink implant, syndicate encryption key (radio implant still kepy in), Mulligan, dart pistol
  - Added: Bluespace compression kit, Karate scroll, EMP kit, binary key
- Syndi-Kit Tactical (A) changes:
  - "Screwed" (syndibomb/powersink), "Lord Singulo" and "Hacker" kits get an energy dagger as they were lacky in any personal self-defence options. 
  - Recon kit gets a Karate scroll.
- Syndi-Kit Special (B) changes:
  - Bond kit gets 1 hollowpoint mag, 1 armor-piercing, and 1 standard 10mm instead of 2x standard 10mm. Also gets a Syndi encryption key
- Also for nukies Syndicate Detonator decreased from 3 to 1 TC since they already get one for free so it's never bought at 3 TC
- Stimpack now uses Pump-Up instead of Stimulants. This makes you immune to damage slowdown, recover 80% faster from stuns/slips/knockdowns and gain 80 stamina regen. Also makes you jitter quite a bit.

## Why It's Good For The Game
Encourages a greater variety of traitor loadouts, which keeps rounds from feeling repetitive.

## Changelog
:cl:
balance: Lots of traitor item costs rebalanced.
balance: Taeclowndo shoes now usable by anyone, but still can only be bought by Clown if not discounted.
balance: Stimpack now uses Pump-Up instead of Stimulants. This makes you immune to damage slowdown, recover 80% faster from stuns/slips/knockdowns and gain 80 stamina regen. Also makes you jitter quite a bit. 
balance: Player minimum requirement for No-Slips removed. 
balance: Contract Kit has it's set of random items tweaked. 
baalnce: Syndi-Kit Tactical also gets an energy dagger or Karate scroll in several of their kits.
balance: Syndi-Kit Special "Bond" kit gets a diversity of magazines and a syndi encryption key.
/:cl:

### CODE REVIEW

**Comment by nikk735** [comment]
i agree with sleepypen 100% very op and synd bomb but posion kit costs enough as it is and there isnt a good use for most the poisons and its inconsistent. no one ever buys posion kit

**Comment by zeskorion** [comment]
> i agree with sleepypen 100% very op and synd bomb but posion kit costs enough as it is and there isnt a good use for most the poisons and its inconsistent. no one ever buys posion kit

i'm gonna change it up soon, remove the dumb shit and replace it with good shit

**Comment by zeskorion** [comment]
also, victor, could you lower esword cost back down to 8? it was increased for the reason "it's popular", which was lame

**Comment by VictorVow** [comment]
Good idea, it is a lot rarer nowadays.

**Comment by eLeCtrOssSnake** [comment]
This is some crazy uncontrolled stuff being thrown, I am going to stay negative about these blind changes. I don't like dice rolls.

**Comment by eLeCtrOssSnake** [comment]
Also, @Victor239 fix your changelog, you have misspelt words.

**Comment by eLeCtrOssSnake** [comment]
I'd suggest to really break down those changes in different pull requests, it's just a bit of a heavy one.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1489,7 +1488,7 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	desc = "Stimpacks, the tool of many great heroes, make you nearly immune to stuns and knockdowns for about \
 			5 minutes after injection."
 	item = /obj/item/reagent_containers/hypospray/medipen/stimulants
-	cost = 5
+	cost = 7
```
Comment (Trilbyspaceclone) [inline code/modules/uplink/uplink_items.dm]: What an odd way to balance healing chems

**Comment by Isy232** [comment]
Stimpen is really REALLY broken, it lasts for 5 whole minutes, gives you basic healing, near stamina damage immunity and super speed.

I think it should only inject like 10 units but cost 2 or 3 TC so its not so OP.

Stimpacks as they are heavily encourage murderboning so you can make the most out of your 5 minutes of godly strength.

I can PR this later if I ever get around to it if you do not want to change this PR to that.

**Comment by qwertyquerty** [comment]
@victor239 this done being balanced?

**Comment by VictorVow** [comment]
@qwertyquerty not quite yet, gonna think about reworking stimpacks.

**Comment by VictorVow** [comment]
Stimpacks are now filled with Pump-Up instead of Stimulants. Pump-Up is a chemical which gives the user 80 stamina regen (for comparison adrenals/stimulants are 35), 80% reduction in stun/slip/knockdown duration, makes you immune to damage slowdown and sleep but makes you jitter quite a bit. Costs 5 TC again. 

This better focuses stimpacks as something you take before engaging on a crusade against the station but doesn't work just the same as using your whole adrenal implant at once.

**Review by ike709 — APPROVED** [review]

**Review by qwertyquerty — APPROVED** [review]

**Pull request merged**